proto/rill/runtime/v1/queries.proto

```diff
@@ -279,7 +280,8 @@ message MetricsViewAggregationRequest {
   TimeRange time_range = 12;
   google.protobuf.Timestamp time_start = 6; // Deprecated in favor of time_range
   google.protobuf.Timestamp time_end = 7; // Deprecated in favor of time_range
-  MetricsViewFilter filter = 8;
+  Expression filter = 8;
```

Comment:
Need to add support for `HAVING` clauses (different from `WHERE` clauses).

Maybe we refactor to `Expression having` and `Expression where`? Or to avoid that, could keep `Expression filter` and add `Expression having` or `Expression measure_filter`?

Reply:
Ya i think separating will keep it cleaner. Combined filter was not clean in the code.

Reply:
Also `having` makes more sense. We can think of using `where` for the main filter in the near future.